setup.py

```diff
@@ -147,6 +147,7 @@ def __init__(self,args, thirdparty_methods):
         self.thirdparty_methods  = thirdparty_methods
         self.with_openmp = args[0].with_openmp
         self.sundials_with_msvc = False
+        self.msvcSLU = False
```

Comment:
This file is poorly documented in general, but since we are adding new _options_ here it would be good with a comment that mentions the effect of this variable, i.e. what happens if this is `True` or `False` during build.

Reply:
This is not really an option that was added, but rather I simply added this line, since this value was missing a default value, since it is only initialized within a function.